opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -1056,25 +1069,31 @@ namespace Opm {
                         const auto& intQuants = *ebosSimulator_.model().cachedIntensiveQuantities(c, /*timeIdx=*/0);
                         const auto& fs = intQuants.fluidState();
                         const double hydrocarbon = fs.saturation(FluidSystem::oilPhaseIdx).value() + fs.saturation(FluidSystem::gasPhaseIdx).value();
-                        hcpv[region] += pv[c] * hydrocarbon;
-                        pres[region] += pv[c] * fs.pressure(FluidSystem::oilPhaseIdx).value();
+                        const double pv =
+                            ebosSimulator_.model().dofTotalVolume(c)
+                            * ebosSimulator_.problem().porosity(c);
+                        hcpv[region] += pv * hydrocarbon;
+                        pres[region] += pv * fs.pressure(FluidSystem::oilPhaseIdx).value();
                     }
                 }
                 for (int c = 0; c < nc; ++c) {
```

Comment:
> BTW and totally unrelated. It seem we have still not fixed all region summations in parallel runs (e.g. by masking or as you did above by only computing on InteriorNodes. But that fix should be in a separate PR.

that's quite possible. the (sequential) FIP numbers after this PR are consistent, with flow_legacy, though...